test/integration/features/basic.feature

```diff
@@ -54,7 +54,7 @@ Feature: Basic
       And stdout of command "minishift config view --format {{.ConfigKey}}:{{.ConfigValue}}" contains "<property>:<value>"
      When executing "minishift config unset <property>" succeeds
      Then stdout of command "minishift config get <property>" is equal to "<nil>"
-      And JSON config file "config/config.json" does not contain key "<property>"
+      And JSON config file "config/config.json" does not have key "<property>" with value assigned
```

Comment:
this sentence reads weird...

`does not have a value assigned to key "<property>"`